website/docs/r/domain.md

```diff
@@ -31,9 +31,9 @@ The following arguments are supported:
 
 In addition to the above arguments, the following attributes are exported:
 
-* `creation_time`
+* `creation_time` - Creation time of the domain.
 
-* `etag`
+* `etag` - ETag of the resource.
```

Comment:
The above was copied from the URL:
https://github.com/googleapis/google-api-go-client/blob/master/admin/directory/v1/admin-gen.go#L2184